app/forms/declaration/BorderTransport.scala

```diff
@@ -101,4 +101,21 @@ object TransportCodes {
       EuropeanVesselIDNumber,
       NameOfInlandWaterwayVessel
     )
+
+    val cash = "A"
+    val creditCard = "B"
+    val cheque = "C"
+    val other = "D"
+    val eFunds = "H"
+    val accHolder = "Y"
+    val notPrePaid = "Z"
```

Comment:
My propose to start with capital letter. It's const, e.g. Cash, CreditCard etc

Reply:
not using them directly that is the reason ideally all the consts would be moved and be config